UM/Trust.py

```diff
@@ -452,17 +522,22 @@ def signedFolderCheck(self, path: str) -> bool:
                 if storage_json:
                     for entry in storage_json:
                         try:
-                            # If this doesn't raise an exception, it's correct.
+                            # If this doesn't raise an exception, it's correct, since central storage uses hashes.
                             central_storage_path = CentralFileStorage.retrieve(entry[1], entry[3], Version(entry[2]))
 
+                            # File could have been removed during execution (also helps with tests).
+                            if not os.path.exists(central_storage_path):
+                                continue
+
                             # If a directory was moved, add all the files in that directory to the file_count. For
                             # individual files mentioned in the central_storage.json increment the file_count by 1.
                             if os.path.isdir(central_storage_path):
                                 file_count += sum([len(files) for _, _, files in os.walk(central_storage_path)])
                             elif os.path.isfile(central_storage_path):
                                 file_count += 1
-                        except (EnvironmentError, IOError):
-                            self._violation_handler(f"Centrally stored file '{entry[1]}' didn't match with checksum or it could not be found")
+                        except (EnvironmentError, IOError, OSError):
```

Comment:
`OSError` is an alias of `EnvironmentError` so this change does nothing.

Reply:
Resolved in 6f7dadba